### DISCUSSION TITLE
I've spen 6 hours trying to fix "No module named 'selenium.webdriver.chromium'". Any suggestions?

### DISCUSSION THREAD

**Discussion by disposablecarcass** [ultrafunkamsterdam/undetected-chromedriver#1695] [Q&A]
I made some changes to my script today - thought I'd try out a newer browser version and see if I could get it to work. I ran into some issues and decided to revert back. I had written down my old undetected-chromedriver version (3.4.6) and Selenium version (3.9.0) so I thought I could just revert back to my old browser and reinstall those versions. I tried reverting back and keep getting this error: 
```

Traceback (most recent call last):
  File "", line 6, in <module>
    import undetected_chromedriver as uc
  File "Cvenv\lib\site-packages\undetected_chromedriver\__init__.py", line 42, in <module>
    from .options import ChromeOptions
  File "site-packages\undetected_chromedriver\options.py", line 8, in <module>
    from selenium.webdriver.chromium.options import ChromiumOptions as _ChromiumOptions
ModuleNotFoundError: No module named 'selenium.webdriver.chromium'
```
In short, I guess I'm asking what configuration of versions for selenium, undetected-chromedriver should work for browser version 112 of Chrome?
Any help is greatly appreciated. Thank you 

@ultrafunkamsterdam 
@mdmintz
@sebdelsol
